uperf-wrapper/uperf-wrapper.py

```diff
@@ -211,13 +211,26 @@ def main():
         server_node = os.environ["server_node"]
     if "client_node" in os.environ:
         client_node = os.environ["client_node"]
+    if "bo_ip" in os.environ:
+        redis_host = os.environ["bo_ip"]
+    if "redis_port" in os.environ:
+        redis_port = os.environ['redis_port']
+    if "redis_collection" in os.environ:
+        redis_collection = os.environ['redis_collection']
```

Comment:
I would make redis_collection env var lowercase/uppercase to make code more robust.